flixel/animation/FlxAnimationController.hx

```diff
@@ -777,6 +777,55 @@ class FlxAnimationController implements IFlxDestroyable
 		play(AnimName);
 		return AnimName;
 	}
+		
+	public function animationList():Array<FlxAnimation>
+	{
+		var arrayNames:Array<FlxAnimation> = [];
```

Comment:
`arrayNames` is an incorrect name

Reply:
I don't think anyone will see that but I changed the name var just in case lol
Resolved